### PR TITLE
Adding random alien

### PR DESCRIPTION
**Pull request by jeremyf**
Example:

```console
$ swnt new alien
Body              :  Insectile, beetle-like, spiderish, wasp-like
Lense             :  Tribalism
Social Structure  :  Multipolar Competetive
```

From Stars Without Number Free Edition p203-207

### CODE REVIEW

File: cmd/alien.go [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+)
+
+// cultureCmd represents the culture command
+var alienCmd = &cobra.Command{
+	Use:   "alien",
+	Short: "Generate an Alien",
+	Long:  ``,
+	Run: func(cmd *cobra.Command, args []string) {
+
+		fmt.Fprintf(tw, alien.New().String())
```
Comment (nboughton) [inline cmd/alien.go]: This should probably read a little differently. fmt.Fprintf is a formatted print to a IO writer. In this case tw being the IO writer, tw is a tabwriter in the Go std lib. It's globally declared in cmd/new.go 

If we're just outputting a single line we probably want a newline afterwards. That can be achieved better with 

    fmt.Fprintln(tw, alien.New())

The .String() call is unnecessary as the struct satisfies the Stringer interface so Go's fmt package will automatically invoke its String method.

**Review by nboughton — CHANGES_REQUESTED** [review]
We should tidy up use of the fmt package here.

**Comment by jeremyf** [comment]
@nboughton I made changes that you recommended.

**Review by nboughton — APPROVED** [review]
Looks good.

**Pull request merged**